homeassistant/components/jellyfin/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""The Jellyfin integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.jellyfin.config_flow import (
+    CannotConnect,
+    InvalidAuth,
+    validate_input,
+)
+from homeassistant.components.jellyfin.const import DATA_CLIENT, DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
The integration does not support YAML.
https://developers.home-assistant.io/docs/configuration_yaml_index/#config_schema

Reply:
Fixed